src/Ng/Jobs/DB2CatalogJob.cs

```diff
@@ -9,39 +9,42 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
+using Ng.Helpers;
 using NuGet.Protocol;
 using NuGet.Services.Configuration;
 using NuGet.Services.Metadata.Catalog;
 using NuGet.Services.Metadata.Catalog.Helpers;
 using NuGet.Services.Metadata.Catalog.Persistence;
+using NuGet.Services.Sql;
 using Constants = NuGet.Services.Metadata.Catalog.Constants;
 
 namespace Ng.Jobs
 {
-    public class Feed2CatalogJob : LoopingNgJob
+    public class DB2CatalogJob : LoopingNgJob
     {
-        private const string FeedUrlFormatSuffix = "&$top={2}&$select=Id,NormalizedVersion,Created,LastEdited,Published,LicenseNames,LicenseReportUrl&semVerLevel=2.0.0";
-
         protected bool Verbose;
-        protected string Gallery;
+        protected ISqlConnectionFactory GalleryDbConnection;
+        protected PackageContentUriBuilder PackageContentUriBuilder;
+        protected IGalleryDatabaseQueryService GalleryDatabaseQueryService;
         protected IStorage CatalogStorage;
         protected IStorage AuditingStorage;
         protected IStorage PreferredPackageSourceStorage;
         protected DateTime? StartDate;
         protected TimeSpan Timeout;
-        protected int Top;
+        protected int Top = 20;
```

Comment:
can this be a const?

Reply:
IMO it should be configurable--20 is a very arbitrary number and I think we could get slightly better perf out of the pipeline if we increased it--but that's out of scope of this PR.

Reply:
Made it configurable using commandline argument `-cursorSize`. Defaults to `20`. 
See https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/8a0b180295c72ae523d1cd4e78db8716565759c7